src/main/java/serenitylabs/tutorials/vetclinic/collections/katas/APetHotel.java

```diff
@@ -1,42 +1,54 @@
 package serenitylabs.tutorials.vetclinic.collections.katas;
 
-import com.google.common.collect.ImmutableList;
-import serenitylabs.tutorials.vetclinic.Breed;
 import serenitylabs.tutorials.vetclinic.Pet;
 
-import java.util.List;
-import java.util.Random;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.TreeSet;
+
+import static java.util.Comparator.comparing;
 
 /**
  * A utility class to generate pet hotels with pets already booked
  */
 public class APetHotel {
-    public static PetAdder with(int petCount) {
-        return new PetAdder(petCount);
-    }
 
-    public static class PetAdder {
-        private final int petCount;
+	private static String name;
+	private Set<Pet> pets = new TreeSet<>(comparing(Pet::getName));
+	
+	public String getName() {
+		return name;
+	}
+
+	public Set<Pet> getPets() {
+		return pets;
+	}
+	
+    public APetHotel(String name) {
+         this.name = name;
+    }
 
-        public PetAdder(int petCount) {
+	
 
-            this.petCount = petCount;
-        }
+	public APetHotel( Set<Pet> pets2 ) {
```

Comment:
Even in test code, I prefer to avoid variable names with number suffixes; maybe something like `newPets` instead?